404.php

```diff
@@ -23,4 +23,14 @@
 		</main><!-- #main -->
 	</div><!-- #primary -->
 
-<?php get_footer();
+	<div id="secondary" class="col-sm-12 sidebar-container" role="complementary">
+		<div class="widget-area">
+			<?php dynamic_sidebar( 'sidebar-404' ); ?>
+		</div><!-- .widget-area -->
+	</div><!-- #secondary -->
+
+<script type="text/javascript">
+	jQuery('.search-field').attr('placeholder', 'lsdev.biz')
```

Comment:
@viscosho please never use inline JavaScript in our core software. On child theme is also not recommended, but I understand sometimes it's necessary. But here we need follow the best practices and in that case, we need to move this code to the LSX core JavaScript file. In this JavaScript, please test if you are on the 404 page before running this line of code.